lib/localize.js

```diff
@@ -177,6 +219,13 @@ export const getLocalizeClass = (superclass = class {}) => class LocalizeClass e
 
 		const allResources = {};
 		const resolvedLocales = new Set();
+
+		localizeResources.forEach((entry, index) => {
```

Comment:
Change 3: because of change 2, some of these resources can be `undefined` -- so we filter them out.

Reply:
No idea about the efficiency but could this be a filter or reduce instead?

Reply:
Like the `flat` above could probably just be a reduce that does both.